content/en/docs/tasks/policy-enforcement/control-headers/index.md

```diff
@@ -1,10 +1,17 @@
 ---
-title: Control Headers and Routing
+title: Control Headers and Routing (Deprecated)
 description: Shows how to modify request headers and routing using policy adapters.
 weight: 20
 keywords: [policies,routing]
 ---
 
+{{< warning >}}
+The mixer policy is deprecated in Istio 1.5 and not recommended for production usage.
+
+Consider using Envoy `Ext-Auth`, `lua` filter, or write a filter using the
+[`Envoy-wasm` sandbox](https://github.com/envoyproxy/envoy-wasm/tree/master/test/extensions/filters/http/wasm/test_data).
```

Comment:
I would advise against using direct links to github master. They almost certainly will break at some point in the future.

Reply:
I copied the link from @mandarjog, there is no release-1.5 branch in that repo, do you have any alternative we can use here?

Reply:
@jplevyak @PiotrSikora is there any public doc about Wasm in Envoy except some random github links?